docs/reference/mapping/params/norms.asciidoc

```diff
@@ -11,11 +11,10 @@ don't need scoring on a specific field, you should disable norms on that
 field. In  particular, this is the case for fields that are used solely for
 filtering or aggregations.
 
-TIP: The `norms` setting must have the same setting for fields of the
```

Comment:
This note seems redundant with the section below ("Norms can be disabled..."), maybe we could remove it?

Reply:
I was thinking of removing it, but the downside would be that the bit of info wouldn't stand out anymore. As a TIP, it would be more visible.

Reply:
I don't have a strong preference, happy to merge as-is. For me it's not such an important point that it needs to be called out in an infobox.